src/components/common/Chart/Chart.tsx

```diff
@@ -121,7 +121,6 @@ function ChartContent(props: ChartContentProps): JSX.Element {
               barThickness,
               backgroundColor: ds.color ?? colors,
               minBarLength: 3,
-              stack: index.toString(),
```

Comment:
Thought this was necessary for the bar groups (from looking at an example), but they work fine without, and it was breaking the pie chart.